recipes/anaconda-client/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "anaconda-client" %}
+{% set version = "1.5.0" %}
+{% set checksum = "e32806844a78f8fb6fa645f14992d4fab10d9bbb5571af2175ed03c5ae3baf28" %}
+
+package:
+  name: {{ name.lower() }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/Anaconda-Platform/anaconda-client/archive/{{ version }}.tar.gz
+  sha256: {{ checksum }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  entry_points:
+    - anaconda = binstar_client.scripts.cli:main
+    - binstar = binstar_client.scripts.cli:main
+    - conda-server = binstar_client.scripts.cli:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - clyent
+    - requests >=2.0
+    - pyyaml
+    - python-dateutil
+    - pytz
+
+  run:
+    - python
+    - clyent
+    - requests >=2.0
+    - pyyaml
+    - python-dateutil
+    - pytz
+
+test:
+  imports:
+    - binstar_client
+    - binstar_client.commands
+    - binstar_client.inspect_package
+    - binstar_client.inspect_package.tests
+    - binstar_client.mixins
+    - binstar_client.scripts
+    - binstar_client.tests
+    - binstar_client.utils
+    - binstar_client.utils.notebook
+    - binstar_client.utils.notebook.tests
+
+  commands:
+    - anaconda --help
+    - binstar --help
+    - conda-server --help
+
+about:
+  home: https://github.com/Anaconda-Platform/anaconda-client
+  license: ???
```

Comment:
Upstream has [confirmed]( https://github.com/Anaconda-Platform/anaconda-client/issues/340#issuecomment-234615466 ) this is BSD 3-Clause. Will update.